trace-sdk/src/main/java/io/bitrise/trace/utils/log/LogMessageConstants.java

```diff
@@ -38,6 +38,9 @@
     public static final String FAILED_TO_READ_SYSTEM_CPU_STATS = "Failed to read System CPU stats, Trace may not " +
             "report any CPU stats for this session.";
     public static final String INITIALISING_SDK = "Initialising the Trace SDK";
+    public static final String METRIC_HEADERS_MISSING = "Metric response headers missing, failed to complete header validation.";
+    public static final String METRIC_HEADERS_MATCH = "Metric response headers match: %d sent and %d received.";
```

Comment:
nit: I think the number will be always the same in this case, so it will be redundant.

Reply:
Updated to remove the second number - now it just contains how many were sent.